src/dashmips.ts

```diff
@@ -78,7 +108,8 @@ export class DashmipsDebugClient extends EventEmitter {
     public call(method: 'verify_breakpoints', params: DashmipsBreakpointInfo[]): void
     public call(method: DebuggerMethods, params?: any[]): void {
         params = params ? params : []
-        this.websocket.send(JSON.stringify({ method, params }))
+        var message = JSON.stringify({ method, params })
```

Comment:
nit: let's use `const` or `let` for out variable declarations wherever we can, [if you're curious as to why](https://alligator.io/js/var-let-const/)

Reply:
Thanks for the resource-- changed from var to const.